ENIGMAsystem/SHELL/Platforms/General/PFwindow.h

```diff
@@ -70,7 +70,7 @@ enum {
   mb_none   =  0,
   mb_left   =  1,
   mb_right  =  2,
-  mb_middle =  3,
+  mb_middle =  3
```

Comment:
This is a pretty standard style artifact; it prevents unnecessary changed lines when adding an item. What was this hurting?

Reply:
Blame says he did it in 0faa0aab969f43204ccdf79572fd9444cc8a22b0

I did not realize what you just said myself, so I will go back in and readd the commas.